src/api/objects/ObjectAPI.js

```diff
@@ -392,6 +405,17 @@ export default class ObjectAPI {
         });
     }
 
+    async #getCurrentUsername() {
+        const user = await this.openmct.user.getCurrentUser();
+        let username = 'Unknown';
```

Comment:
Let's just leave this undefined if the user is unknown. All of our legacy objects will have undefined users, and we have logic for handling that case in the Vue component.